standard/expressions.md

```diff
@@ -6814,7 +6814,7 @@ A constant expression shall either have the value `null` or one of the following
 Only the following constructs are permitted in constant expressions:
 
 - Literals (including the `null` literal).
-- References to `const` members of class and struct types.
+- References to `const` members of class, struct, and interface types.
```

Comment:
I'm finding dozens of hits for "class _or_ struct" in other places that should now expand to "class, struct, or interface."

Reply:
upcoming commit. I searched for all instances of "class or struct" and where appropriate replace with "class, interface, or struct". Many locations didn't require a change because of context (instance constructor, instance fields, etc)